models/quickbooks__profit_and_loss.sql

```diff
@@ -7,7 +7,7 @@ with general_ledger_by_period as (
 
 final as (
     select
-        period_first_day as calendar_date,
+        period_first_day as calendar_date, --  Slated to be deprecated; we recommend using `period_first_day` or `period_last_day`
```

Comment:
I just noticed that period_first_day and period_last_day aren't included in the quickbooks__profit_and_loss SQL even though the yml definition says the columns exist.

Reply:
Ah @sbrudz you're right, this was mistakenly removed during the PR. We will have this fix ready first thing next week!

Reply:
Thank you!

Reply:
Thanks @sbrudz  for catching this, the package has been updated!